processor/preverify/preverify.go

```diff
@@ -30,8 +30,18 @@ type Config struct {
 	// nil or empty implies []int{http.StatusOK}, which is considered to be
 	// the current best practice.
 	GoodStatusCodes []int
+
+	// MaxContentLength
+	//
+	// Zero implies DefaultMaxContentLength, and a negative number implies
+	// "unlimited."
```

Comment:
Might better to also document "negative implies unlimited" behavior in the description  of flagMaxContentLength. Users may sometimes want to run the packager without the size limit.

Reply:
Good point. Done.